packages/data-models/constants.ts

```diff
@@ -35,6 +35,11 @@ export const APP_FIELDS = {
   DEPLOYMENT_NAME: `${FIELD_PREFIX}._deployment_name`,
 };
 
+export const APP_LANGUAGES = {
+  /** Language used during first load. If translations do not exist will default to source strings (en) */
```

Comment:
```suggestion
  /** Language used during first load. If translations do not exist will default to source strings (gb_en) */
```

Reply:
I'm not sure what "generic" English would be. Since the app content always uses British spelling I think it's clearer to call this `gb_en`

Reply:
I think I was trying to distinguish between languages that we actually have specific overrides for vs just using the _source_ strings. So equally I could have just said the first language was _default_ or _source_, but I have no issue with calling it gb_en